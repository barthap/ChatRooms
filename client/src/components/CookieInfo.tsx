import React from 'react';

export default function CookieInfo() {
  return (
    <details>
      <summary>This webiste uses cookies to store user session, but who cares ðŸ˜‡</summary>
      <small>
        If you seriously don't agree for cookies (but they're really yummy, trust me), just leave
        this website, clear cookies in the browser settings (or wait 24h, then they're expire
        anyway) and never enter this website again, but you'll probably regret. Stupid UE law.
      </small>
    </details>
  );
}
