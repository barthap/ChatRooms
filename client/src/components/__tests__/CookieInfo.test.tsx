import { render } from '@testing-library/react';
import React from 'react';

import CookieInfo from '../CookieInfo';

describe('<CookieInfo />', () => {
  it('renders correctly', () => {
    const cookieInfo = render(<CookieInfo />);
    expect(cookieInfo).toMatchSnapshot();
  });
});
