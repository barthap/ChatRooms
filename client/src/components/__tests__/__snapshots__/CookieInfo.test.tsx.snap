// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CookieInfo /> renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <details>
        <summary>
          This webiste uses cookies to store user session, but who cares ;)
        </summary>
        <small>
          If you seriously don't agree for cookies (but they're really yummy, trust me), just leave this website, clear cookies in the browser settings (or wait 24h, then they're expire anyway) and never enter this website again, but you'll probably regret. Stupid UE law.
        </small>
      </details>
    </div>
  </body>,
  "container": <div>
    <details>
      <summary>
        This webiste uses cookies to store user session, but who cares ;)
      </summary>
      <small>
        If you seriously don't agree for cookies (but they're really yummy, trust me), just leave this website, clear cookies in the browser settings (or wait 24h, then they're expire anyway) and never enter this website again, but you'll probably regret. Stupid UE law.
      </small>
    </details>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
